charts/prometheus-thanos/README.md

```diff
@@ -233,9 +233,8 @@ The following table lists the configurable parameters of the prometheus-thanos c
 | `storeGateway.image.repository` | Docker image repo for store gateway | `quay.io/thanos/thanos` |
 | `storeGateway.image.tag` | Docker image tag for store gateway | `v0.10.1` |
 | `storeGateway.image.pullPolicy` | Docker image pull policy for store gateway | `IfNotPresent` |
-| `storeGateway.serviceAccount.create` | Create service account | `true` |
-| `storeGateway.serviceAccount.annotations` | Service account annotations | `nil` |
-| `storeGateway.indexCacheSize` | Index cache size | `500MB` |
+| `storeGateway.indexCacheConfig` | Type of the index cache, either `IN-MEMORY` or `MEMCACHED` | `IN-MEMORY` |
```

Comment:
values & description for config & type seem twisted.

Reply:
Thanks, updated. 👍